cli/js/net.ts

```diff
@@ -9,11 +9,10 @@ export type Transport = "tcp";
 // TODO support other types:
 // export type Transport = "tcp" | "tcp4" | "tcp6" | "unix" | "unixpacket";
 
-// TODO(ry) Replace 'address' with 'hostname' and 'port', similar to DialOptions
-// and ListenOptions.
 export interface Addr {
   transport: Transport;
```

Comment:
I'm not entirely sure that `transport` should be part of `Addr`....

Reply:
@ry I suggest to remove `Addr.transport` and make it a property of `Conn`:

```ts
export interface Addr {
   hostname: string, // <-- not sure about it either because we return ip4/ip6 right now
   port: number
}

interface Conn {
   transport: string;
   tls: boolean;
   localAddr: Addr;
   remoteAddr: Addr;

   ...
}
```